examples/zkapps/04-zkapp-browser-ui/contracts/src/Add.test.ts

```diff
@@ -19,31 +20,36 @@ import {
 function createLocalBlockchain() {
   const Local = Mina.LocalBlockchain();
```

Comment:
nit: since this is a set of unit tests, I'd probably turn off proving with `proofsEnabled: false` here

Reply:
Yeah I was thinking about this. I very muich agree in the context of a unit test but this is also an example. I wonder if too much test isolation comes at the expense of illustrating the system. wdyt?

Reply:
I think some of the other tutorials, where code examples aren't Jest tests, may be better places to show real compile / prove